### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 22

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#198] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152254008-f535e8eb-9f1c-4a49-8cb4-71ce07d1501f.png)

**Comment by Elia-Belli** [comment]
**(1)** Il primo punto è semplice, si prende un frutto di ciascun tipo e si distribuiscono i 6 rimanenti senza vincoli:

$\binom{6+9-1}{6}$

**(2)** Non sono sicuro la soluzione di @RiaMeLa per il secondo punto sia corretta.

In un problema del genere (scrittura additiva) col vincolo sul massimo bisogna applicare il PIE sul numero di frutti (variabili), ovvero un PIE a 9 termini in questo caso, per ottenere il complemento della richiesta ("almeno un frutto va preso 2 volte"):

$\\# \bigcup_{1-9} i$ =#A + #B +#C - (#intersezioni 2 a 2) + (#intersezioni a 3) - (#intersezioni a 4) + (#intersezioni a 5) - (#intersezioni a 6) + (#intersezioni a 7) - (#intersezioni a 8) + (#intersezioni a 9)

Poi si procede sottraendo $\\# \bigcup_{1-9} i$ da tutte le combinazioni con ripetizione senza vincoli:

Soluzione = $\binom{15+9-1}{9} - \\# \bigcup_{1-9} i$
